test/Ported/Attachments/AttachmentsRevisionsTest.ts

```diff
@@ -0,0 +1,253 @@
+
+import * as assert from "assert";
+import { testContext, disposeTestDocumentStore } from "../../Utils/TestUtil";
+
+import {
+    DeleteDocumentCommand,
+    DocumentStore, GetStatisticsOperation, IDocumentSession,
+    IDocumentStore, IMetadataDictionary,
+} from "../../../src";
+import {User} from "../../Assets/Entities";
+import {PutAttachmentOperation} from "../../../src/Documents/Operations/Attachments/PutAttachmentOperation";
+import {CONSTANTS} from "../../../src/Constants";
+
+describe("AttachmentsRevisions", function () {
+
+    let store: IDocumentStore;
+
+    beforeEach(async function () {
+        store = await testContext.getDocumentStore();
+    });
+
+    afterEach(async () =>
+        await disposeTestDocumentStore(store));
+
+    it("can put attachments", async () => {
+        await testContext.setupRevisions(store, false, 4);
```

Comment:
can we `opts` it here? Can't tell what's false and 4.

Reply:
just install good IDE: 

![image](https://user-images.githubusercontent.com/1250431/46062549-71847380-c16a-11e8-98e0-350cf1725da9.png)


;-)